platform/mellanox/iproute2/Makefile

```diff
@@ -0,0 +1,49 @@
+SHELL = /bin/bash
+.ONESHELL:
+.SHELLFLAGS += -e
+
+MAIN_TARGET = iproute2-mlnx_$(IPROUTE2_MLNX_VERSION_FULL)_$(CONFIGURED_ARCH).deb
+IPROUTE2_DEB = iproute2_$(IPROUTE2_MLNX_VERSION_FULL)_$(CONFIGURED_ARCH).deb
+IPROUTE2_TMP := $(shell mktemp -u -d -t ipr.XXXXXXXXXX)
+
+$(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
+	# Remove any stale files
+	rm -rf iproute2-$(IPROUTE2_MLNX_VERSION)
+
+	wget -O iproute2_$(IPROUTE2_MLNX_VERSION).orig.tar.xz http://deb.debian.org/debian/pool/main/i/iproute2/iproute2_$(IPROUTE2_MLNX_VERSION).orig.tar.xz
```

Comment:
Can we use dget instead of wget? Please refer the PR below
``` https://github.com/sonic-net/sonic-buildimage/pull/13517 ```

Reply:
Done